app/src/components/Grid.svelte

```diff
@@ -41,6 +76,21 @@
   <rect width="100%" height="100%" fill="url(#grid)" />
 </svg>
 
+<!-- this is pretty slow . todo: canvas api? :3c -->
```

Comment:
Remove comment about Canvas API. It's not a todo! Your DOM implementation is perfect for this game 🔥🙏 (same as the other comment)